scripts/bootstrap.py

```diff
@@ -110,6 +111,7 @@ def main():
         marathon_jenkins_port = os.environ['PORT1']
         mesos_master = os.environ['JENKINS_MESOS_MASTER']
         ssh_known_hosts = os.environ['SSH_KNOWN_HOSTS']
+        marathon_name = os.environ['MARATHON_NAME']
```

Comment:
This will fail without an accompanying change to Universe. Maybe this should be `os.getenv('MARATHON_NAME', 'marathon')` and set a default to prevent failures.

Reply:
It's the only thing I'm adding in the new Universe release and should be there forever after that. Still think it's worth setting the default in code? Just seemed odd to me being the only default set in code.